CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ if(NOT CMAKE_BUILD_TYPE)
         "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
+option(COMPILE_PYTHON3_BINDINGS "Enable/Disable Python bindings. Python bindings are disabled by default." OFF)
+option(BUILD_TESTING "Enable/Disable test applications." OFF)
```

Comment:
The `BUILD_TESTING` variable is already defined when including Ctest:
https://cmake.org/cmake/help/v3.30/module/CTest.html